src/shared/lib/NumberUtils.ts

```diff
@@ -6,3 +6,8 @@ export function toConditionalPrecision(number: number, precision: number, thresh
 
     return number.toPrecision(precision);
 };
+
+export function floatValueIsNA(number: number): boolean {
+    const minFloatValue: number = 1.0e-44;
+    return (number > 0 && number < minFloatValue);
```

Comment:
one caution : floatValueIsNA(null) is going to return false because null > 0 returns false
Up above in @computed private get hasCcfMCopies(), it looks for any value where !floatValueIsNA ... so if there is a null present in the column, it would appear that there *was* something present (something not NA). Maybe the source data is always initialized to non-null values, but otherwise you may want to test for null above (or here) and do the right thing.